fixtures/american-dj/mini-dekker.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Dekker",
+  "shortName": "ADJ Mini Dekker",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["David"],
+    "createDate": "2023-12-06",
+    "lastModifyDate": "2023-12-06"
+  },
+  "links": {
+    "manual": [
+      "http://adjmedia.s3-website-eu-west-1.amazonaws.com/manuals/Mini%20Dekker_FR%2005152019.pdf"
```

Comment:
This is a French manual dated 5/19.  There's an English one dated 1/2021 at https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/8777/ADJ%20Mini%20Dekker%20-%20User%20Manual.pdf